packages/node-build-scripts/package.json

```diff
@@ -23,8 +23,9 @@
         "cross-spawn": "^7.0.3",
         "css-select": "^5.1.0",
         "dom-serializer": "^2.0.0",
-        "eslint": "^8.57.0",
+        "eslint": "^9.20.0",
         "eslint-formatter-codeframe": "^7.32.1",
+        "eslint-formatter-junit": "^8.3.0",
```

Comment:
Needed to install this formatter since ESLint removed it from the builtin ones: https://eslint.org/docs/latest/use/migrate-to-9.0.0#-removed-multiple-formatters